action.yml

```diff
@@ -39,37 +39,59 @@ runs:
         fi
       shell: bash
 
-    - name: Check for /jira command in PR comments
-      if: github.event_name == 'issue_comment' && github.event.issue.pull_request != null
+    - name: Check for /jira command in PR comments and descriptions
+      if: ${{ github.event_name == 'issue_comment' && github.event.issue.pull_request != null || github.event_name == 'pull_request_target' && github.event.action == 'edited' }}
       env:
         REPOSITORY: ${{ github.event.repository.full_name }}
         PR_NUMBER: ${{ github.event.issue.number }}
         PR_TITLE: ${{ github.event.issue.title }}
         PR_BODY: ${{ github.event.issue.body }}
         PR_AUTHOR: "${{ github.event.issue.user.login }}"
         COMMENT_BODY: ${{ github.event.comment.body }}
+        PR_URL: ${{github.event.issue.url}}
         GITHUB_TOKEN: ${{ inputs.token }}
         JIRA_TOKEN: ${{ inputs.jira_token }}
       run: |
         set -euo pipefail
-        if [[ ! "$COMMENT_BODY" =~ ^/jira-epic ]]; then
-          echo "No recognized slash command found."
+        if [[ "${{ github.event_name }}" == "issue_comment" ]]; then
+          echo "Event: New comment added to pull request."
+        elif [[ "${{ github.event.action }}" == "edited" ]]; then
+          echo "Event: Pull request description updated."
+          # Override variables based on the pull_request event type
+          PR_TITLE=$(echo "${{ github.event.pull_request.title }}")
```

Comment:
I'm pretty sure this is prone to bash injections. I suggest splitting this into 2 steps and only using env variables to pass github variables with user input. Code duplication sucks, but better be safe than sorry.

Reply:
Agreed, I'll re-duplicate!